src/AlgebraicGeometry/ToricVarieties/cohomCalg/VanishingSets/attributes.jl

```diff
@@ -65,14 +65,15 @@ Normal, non-affine, smooth, projective, gorenstein, fano, 2-dimensional toric va
 
 julia> vs = vanishing_sets(dP1)
 3-element Vector{ToricVanishingSet}:
- Toric vanishing set for cohomology index 0
- Toric vanishing set for cohomology index 1
- Toric vanishing set for cohomology index 2
+ Toric vanishing set for cohomology indices [0]
+ Toric vanishing set for cohomology indices [1]
+ Toric vanishing set for cohomology indices [2]
 
-julia> cohomology_index(vs[3])
-2
+julia> cohomology_indices(vs[3])
+1-element Vector{Int64}:
+ 2
 ```
 """
-function cohomology_index(tvs::ToricVanishingSet)
-    return tvs.i::Int
+function cohomology_indices(tvs::ToricVanishingSet)
+    return tvs.cis::Vector{T} where {T <: IntegerUnion}
```

Comment:
Is the abreviation 'cis' really standard or should we better use a longer/clearer form? (Keeping in mind that not only toric geometers are using Oscar).

Reply:
Then I am fine with it.

Reply:
Thank you.